pyproject.toml

```diff
@@ -43,13 +43,14 @@ numpy = [
     {version = ">=1.22", python = ">=3.10"}
 ]
 scikit-learn = ">=0.22"
-lightgbm = ">=2.3, <=3.2.1"
+lightgbm = ">=2.3"
 catboost = ">=0.26.1"
 optuna = "*"
 torch = [
     {platform = "win32", python = "3.6.1", version = "1.7.0"},
     {version = ">=1.0.0"}
 ]
+faiss-cpu="*"
```

Comment:
```suggestion
faiss-cpu={version = "*", optional = true}
```

Reply:
1) add section for hypex requirements:
```
hypex = [
    "faiss-cpu"
]
```
2) add "hypex" to requiremets for "all"

Reply:
Done